dart/math/Geometry.h

```diff
@@ -533,6 +533,35 @@ Eigen::Vector2d computeClosestPointOnSupportPolygon(
     const Eigen::Vector2d& _p,
     const SupportPolygon& _support);
 
+
+// Represents a bounding box with minimum and maximum coordinates.
+class BoundingBox {
```

Comment:
I agree with keeping the name as just ``BoundingBox`` and not saying anything about AABB versus OBB. The class itself is fundamentally just describing the data of a bounding box, and whether it's AABB or OBB will depend on the context in which it's used. In most cases, the Shape that it belongs to will have some arbitrary orientation with respect to the "world" coordinates, so it would really be OBB. In order for it to be strictly AABB, we would need to have a way to compute the bounding box as a function of the shape's orientation.